src/main/java/games/strategy/engine/lobby/server/db/UserController.java

```diff
@@ -45,6 +45,7 @@ public HashedPassword getPassword(final String userName) {
       ps.close();
       return returnValue == null ? null : new HashedPassword(returnValue);
     } catch (final SQLException sqle) {
+      // SEVERE
       logger.info("Error for testing user existence:" + userName + " error:" + sqle.getMessage());
```

Comment:
Probably should change this log message so it isn't a duplicate of the one below.